### PR TITLE
Added JFrog metrics integration

### PR DESCRIPTION
**Pull request by MitaliBo**
### What does this PR do?

This PR is the integration of JFrog Artifactory and Xray metrics integration with Datadog dashboards.

### Motivation

Since Metrics related to JFrog products Artifactory and Xray are being exposed using OpenMetric format, it will be easy for users to visualize the metrics using Datadog dashboards. This integration will provide pre configured dashboards made out of the metrics.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: jfrog_metrics/datadog-agent-with-operator.tar.gz [hunk 1]
```diff
@@ -0,0 +1 @@
+Not Found
```
Comment (coignetp) [inline jfrog_metrics/datadog-agent-with-operator.tar.gz:1]: Can we remove this file?

File: jfrog_metrics/metadata.csv [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+jfrog.xray.sys_memory_used_bytes,gauge,,,,,0,jfrog_metrics,Sys Memory Used Size
+jfrog.xray.sys_cpu_ratio,gauge,,,,,0,jfrog_metrics,Sys CPU Ratio
+jfrog.xray.jfxr_data_artifacts_total,gauge,,,,,0,jfrog_metrics,Total Scanned Artifacts
+jfrog.xray.jfxr_data_components_total,gauge,,,,,0,jfrog_metrics,Total Components Artifacts
+
+
+
+
+
+
```
Comment (coignetp) [inline jfrog_metrics/metadata.csv:34]:
```suggestion
```

File: jfrog_metrics/datadog_checks/jfrog_metrics/check.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+
+class JfrogMetricsCheck(OpenMetricsBaseCheck):
+    """
+    Collect metrics from JFrog
+    """
+
+    def __init__(self, name, init_config, instancetype, instances=None):
```
Comment (coignetp) [inline jfrog_metrics/datadog_checks/jfrog_metrics/check.py:9]:
```suggestion
    def __init__(self, name, init_config, instances=None):
```
Let's have `instancetype` as a configuration option in the `spec.yaml` file

**Review by coignetp — CHANGES_REQUESTED** [review]
Hi, I have a few suggestion regarding the configuration.
I'll sync this integration with staging so we can make sure the dashboards are fine

File: jfrog_metrics/datadog_checks/jfrog_metrics/check.py (hunk 3 above)
Comment (MitaliBo) [reply jfrog_metrics/datadog_checks/jfrog_metrics/check.py]: Sure i have committed necessary changes in spec and tests for instanceType

File: jfrog_metrics/datadog-agent-with-operator.tar.gz (hunk 1 above)
Comment (MitaliBo) [reply jfrog_metrics/datadog-agent-with-operator.tar.gz]: Done

**Comment by MitaliBo** [comment]
> Hi, I have a few suggestion regarding the configuration.
> I'll sync this integration with staging so we can make sure the dashboards are fine
> 
> I aslo made an internal request to have the logo in a tile

Hi I have committed changes regarding your suggestions. Thanks for the logo request. Can you please review, all checks have passed

File: jfrog_metrics/README.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+### What is JFrog Artifactory and Xray
+JFrog Enterprise with Xray features Artifactory Enterprise and Xray. Together, they empower DevOps teams to improve their productivity to increase velocity and deliver high quality software releases with confidence. 
+
+Artifactory supports multiple build packages, artifacts, and metadata. It allows DevOps teams to have the freedom of choice of build packages like Bower, Chef, CocoaPods, Conan, Conda, CRAN, Debian, Docker, Golang, Gradle, Git LFS, Helm, Ivy, Maven, npm, NuGet, Opkg, P2, PHP Composer, Puppet, PyPI, RPM, RubyGems, SBT, Vagrant & VCS, CI/CD platforms, and devops tools.
+
+Artifactory Enterprise supports multi-region, multi-cloud, and hybrid replication for geographically distributed teams allowing you to replicate between a source repository and multiple targets simultaneously and security features such as IP filtering, CNAME, and data encryption at rest.Artifactory supports Kubernetes for microservices and containerized applications. Manage your deployments and gain insight into dependencies using Artifactory as your Kubernetes registry.
```
Comment (apigirl) [inline jfrog_metrics/README.md:11]:
```suggestion
Artifactory Enterprise supports multi-region, multi-cloud, and hybrid replication for geographically distributed teams allowing you to replicate between a source repository and multiple targets simultaneously and security features such as IP filtering, CNAME, and data encryption at rest. Artifactory supports Kubernetes for microservices and containerized applications. Manage your deployments and gain insight into dependencies using Artifactory as your Kubernetes registry. 
```

**Review by apigirl — APPROVED** [review]
just one small typo, otherwise docs look great

**Comment by MitaliBo** [comment]
Closing this one to merge metrics with existing jfrog_platform tile

**Pull request closed without merging**